README.md

```diff
@@ -183,8 +183,14 @@ pip install -e .
 
 ### Running CLI
 
+#### Install Terraform
+Check Instructions here : https://learn.hashicorp.com/tutorials/terraform/install-cli#:~:text=If%20you%20get%20an%20error,directory%20where%20Terraform%20was%20installed.
```

Comment:
Can you replace it with this link please: 
https://learn.hashicorp.com/tutorials/terraform/install-cli
```suggestion
Check Instructions here: https://learn.hashicorp.com/tutorials/terraform/install-cli
```